content/en/docs/concepts/signals/baggage.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Baggage
 weight: 4
+description: Contextual information that is passed between spans
```

Comment:
_between spans_

Not sure if Baggage is tied to Spans in anyway..

Reply:
I would recommend just saying "between signals". Even if it's mostly used with traces today.

Reply:
```suggestion
description: Contextual information that is passed between signals
```